backports/py-grpcio/APKBUILD

```diff
@@ -0,0 +1,49 @@
+# Contributor: Daiki Maekawa <daikimaekawa29@gmail.com>
+# Maintainer: Daiki Maekawa <daikimaekawa29@gmail.com>
+pkgname=py-grpcio
+pkgver=1.20.0
+pkgrel=0
+pkgdesc="Package for gRPC Python."
+url="https://grpc.io/"
+arch="all"
+license="Apache License 2.0"
+depends="py-six py-futures py-enum34"
```

Comment:
better to specify version constraint
```
    "six>=1.5.2",
    "futures>=2.2.0; python_version<'3.2'",
    "enum34>=1.0.4; python_version<'3.4'",
```
python3 on Alpine 3.7 is 3.6.8. futures and enum34 looks needed only for py2. (they should be appended in `_py2()`)